app/assets/stylesheets/views/_homepage.scss

```
@@ -77,14 +77,10 @@
 }
 
 .home-divider {
-  margin: govuk-spacing(6) 0 govuk-spacing(3);
+  margin: govuk-spacing(4) 0;
   clear: both;
   border: 5px solid govuk-colour("black");
   border-width: 5px 0 0;
```

Could we tidy up these two declarations to one?

```suggestion
  border-top: 5px solid govuk-colour("black");
```

---

Included this change but I think we _may_ end up deleting the `hr`s altogether when all the work is merged in (and adding border-top on the section instead). https://gds.slack.com/archives/C025C1QUSL8/p1637751262001000